s3/pkg/service/multipart.go

```diff
@@ -306,6 +327,40 @@ func (s *s3Service) CompleteMultipartUpload(ctx context.Context, in *pb.Complete
 		return err
 	}
 
+	md5Writer := md5.New()
+	var totalSize int64 = 0
+	for i := 0; i < len(in.CompleteParts); i++ {
+		if in.CompleteParts[i].PartNumber != int64(i+1) {
+			log.Infof("wrong order for part number. ")
+			err = ErrInvalidPart
+			return err
+		}
+		part, ok := multipart.Parts[i+1]
+		if !ok {
+			log.Infoln("missed object part. partno:", i)
+			err = ErrInvalidPart
+			return err
+		}
+
+		if part.Etag != in.CompleteParts[i].ETag {
+			log.Infoln("part etag in meta store is not the same with client's part, partno:", i)
```

Comment:
Same here.

Reply:
thanks, will use Errorln instead of infof